notebooks/03-query/10-functions-ip.ipynb

```diff
@@ -185,6 +185,81 @@
     "display.sql(sql)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d23b9a99-cb5e-4b38-b4f9-07aec117ba5e",
+   "metadata": {},
+   "source": [
+    "### New in Druid 29.0.0 - IPV6_MATCH function\n",
```

Comment:
Suggest you just add subheads under "Filtering query results against subnets" - one for IPv4 (the existing examples) and another for IPv6 (your additional SQL examples).

Reply:
done